hermes/src/ontology/mod.rs

```diff
@@ -82,3 +107,27 @@ where
     String::deserialize(deserializer)
         .and_then(|string| base64::decode(&string).map_err(|err| Error::custom(err.to_string())))
 }
+
+#[derive(Debug, Clone, PartialEq, PartialOrd, Deserialize, Serialize)]
+#[serde(rename_all = "camelCase")]
+pub struct LoadMessage {
+    /// The Snips component that was loaded
+    pub component: SnipsComponent,
+    /// Optional id associated to a reload operation of a component
+    pub load_id: Option<String>,
+}
+
+impl<'de> HermesMessage<'de> for LoadMessage {}
+
+#[derive(Debug, Clone, PartialEq, PartialOrd, Deserialize, Serialize)]
+#[serde(rename_all = "camelCase")]
+pub struct SiteLoadMessage {
+    /// The site concerned
+    pub site_id: String,
+    /// Optional id associated to a reload operation of a component
+    pub load_id: Option<String>,
+    /// The Snips component that was loaded
+    pub component: SnipsComponent,
```

Comment:
hum, this may finally not be a good idea, this means a component can post `LoadMessage`s for another component in its channel, this looks like a footgun

Reply:
Yep totally agree on this.